README.md

```diff
@@ -1,11 +1,64 @@
 # BrowserStack GitHub Actions
 
-This respository contains the GitHub Actions to help you integrate your test suite with BrowserStack cloud.
+This respository contains a library of GitHub Actions to help you integrate your test suite with the [BrowserStack](https://browserstack.com) device cloud. 
+
+You need a BrowserStack username and access-key to run your tests on the BrowserStack device cloud. You can [sign-up for free trial](https://www.browserstack.com/users/sign_up) if you do not have an existing account. We love open source projects. If you want to test your open source project on BrowserStack then [sign-up](https://www.browserstack.com/open-source) for your lifetime free access to all our products.
```

Comment:
```suggestion
You need a BrowserStack username and access-key to run your tests on the BrowserStack device cloud. You can [sign-up for free trial](https://www.browserstack.com/users/sign_up) if you do not have an existing account. 
If you want to test your open source project on BrowserStack then [sign-up here](https://www.browserstack.com/open-source) for your lifetime free access to all our products.
```
How about trying to separate out the `open-source` part and focusing on it on a new line ? ...